jekyll/_ccie/troubleshooting-install.md

```diff
@@ -13,6 +13,62 @@ published: true
 This means that the GitHub Enterprise server is not returning the intermediate SSL certificates. Check your GitHub Enterprise instance with <https://www.ssllabs.com/ssltest/analyze.html> - it may report some missing intermediate certs. You can use tools like <https://whatsmychaincert.com/> to get the full certificate chain for your server.
 
 
+#### Docker isn't working?
+
+I would also like to note that there is a bit of extra effort to get docker working natively within container in our single box node. Please see the instructions here to get docker working: https://circleci.com/docs/enterprise/docker-builder-config/#sharing-the-docker-socket
+
+Please note that you'll have to have
+
+`export CIRCLE_DOCKER_RUN_ARGUMENTS="-v /var/run/docker.sock:/var/run/docker.sock"`
+
+in `/etc/circle-installation-customizations` for it to work. You also won't need to have docker specified in the circle.yml as the container will have access to the docker socket in the background.
+
+#### Can I use a custom image in single box mode?
+
+Yes! You can do this by adding this to a file called `/etc/circle-installation-customizations`:
+
+`export CIRCLE_CONTAINER_IMAGE_URI="docker://circleci/build-image:latest"`
+That'll give you the same image that is currently used on circleci.com, but you can replace the URI image with anything that is currently pulled unto the machine or exists in dockerhub. This works with [custom images](https://github.com/circleci/image-builder) as well. 
```

Comment:
Could be nice to reference where to see the current .com container tag, as documented at https://circleci.com/docs/enterprise/docker-install/#2-builder-instances